src/backend/web/local/bootstrap.py

```diff
@@ -132,6 +133,24 @@ def update_event(cls, key: EventKey, auth_token: str) -> None:
         event_awards = cls.fetch_event_detail(key, "awards", auth_token)
         list(map(lambda t: cls.store_award(t, event), event_awards))
 
+        # Get insights from prod API
+        # event_oprs = cls.fetch_event_detail(key, "oprs", auth_token)
+        # cls.store_eventdetail(event, "matchstats", event_oprs)
+
+        # Calculate insights locally
+        matchstats_dict = MatchstatsHelper.calculate_matchstats(
```

Comment:
this bit (and the associated test fixes) should probably be split out into a separate PR we can land first to make your (and my!) life simpler